tests/testdata.py

```diff
@@ -38,6 +38,20 @@
     }
 }
 
+TEST_METADATA_MULTI_ANNON = {
```

Comment:
Instead of duplicating `TEST_METADATA`, can you copy and just override `ANNOTATION_KEY`?